lib/Mojo/IOLoop/ReadWriteProcess/Session.pm

```diff
@@ -43,6 +43,7 @@ sub _protect {
 }
 
 sub enable {
+  return if $singleton->handler();
```

Comment:
Looking through the code it seems that after this change you can only call `session->enable` once, and when `session->disable` has been called afterwards, it cannot be enabled again, since the `handler` attribute is never cleared.

Reply:
right, my fault.

I still would not "renable" unconditional at least not with current code in ReadWriteProcess, case we call enable with each `start()`